training/src/anemoi/training/diagnostics/mlflow/logger.py

```diff
@@ -625,8 +625,8 @@ def finalize(self, status: str = "success") -> None:
 
         super().finalize(status)
 
-    @staticmethod
```

Comment:
Hey Nathan, what's the reason for dropping this decorator? This is used in the mlflow command (https://github.com/ecmwf/anemoi-core/blob/bbd152d434c9388a9dfc9100c7608255d932f79a/training/src/anemoi/training/commands/mlflow.py#L253)